collectors/raw_request.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+class QM_Collector_Raw_Request extends QM_Collector {
+
+	public $id = 'raw_request';
```

Comment:
I was unsure on exactly what to call this collector; "request" would make the most sense, but it's an overloaded term already being used.